pkg/cli.go

```diff
@@ -17,12 +17,16 @@ var (
 
 func NewApp() *cli.App {
 	return &cli.App{
-		Name:    "jumpstart",
-		Usage:   "jumpstart your project",
-		Version: VERSION + "-" + COMMIT + "+" + BUILDTIME,
+		Name:      "jumpstart",
+		Usage:     "jumpstart your project",
+		UsageText: "jumpstart command [command option]",
```

Comment:
We should remove this line and use what the library generates

Reply:
done